### PR TITLE
improve plot (shading, display current time)

### LINKED ISSUE #227: improve plot: add shading to separate different days in the plot and a vertical line to display the current time
(request from Rune)

### LINKED ISSUE #229: Display month name instead of number in plot ("Feb" instead of 2) and always use 24 hour format

### PR DESCRIPTION
**Pull request by KristianKjerstad**
* display current time a line in the plot
* add shading to plot to differentiate different days
* update format of time in plot

image

closes #227 
closes #229

### CODE REVIEW

**Comment by KristianKjerstad** [comment]
@Timothy-Edward-Kendon feel free to take a look at the screenshot and see that the format looks okay

File: web/custom-plugins/forecast-of-response/src/components/Plots/plotUtils.tsx [hunk 1]
```diff
@@ -4,10 +4,19 @@ export const getTime = (
...
+      year: 'numeric',
+      month: 'short',
+      day: 'numeric',
+      hour: '2-digit',
+      minute: '2-digit',
+      second: '2-digit',
+    }
     if (useLocalTimezone) {
-      return new Date(dateString).toLocaleString(navigator.language)
+      return new Date(dateString).toLocaleString('no-NB', formatOptions)
```
Comment (soofstad) [inline web/custom-plugins/forecast-of-response/src/components/Plots/plotUtils.tsx:16]: Any reason to hard code "norway" as local time here?

**Review by soofstad — APPROVED** [review]
One comment

File: web/custom-plugins/forecast-of-response/src/components/Plots/plotUtils.tsx (hunk 1 above)
Comment (Timothy-Edward-Kendon) [reply web/custom-plugins/forecast-of-response/src/components/Plots/plotUtils.tsx:16]: We don't want to hard code Norway as the local time. Operations can occur all over the world.

Comment (KristianKjerstad) [reply web/custom-plugins/forecast-of-response/src/components/Plots/plotUtils.tsx:16]: fixed!

**Pull request merged**